contracts/REXMarket.sol

```diff
@@ -267,7 +271,9 @@ abstract contract REXMarket is Ownable, SuperAppBase, Initializable {
         ISuperToken _inputToken,
         uint256 _rateTolerance,
         ITellor _tellor,
-        uint256 _inputTokenRequestId,
+        address _uniswapPool,
+        address _uniswapInputToken,
+        address _uniswapOutputToken,
         uint128 _affiliateFee,
         uint128 _feeRate
```

Comment:
This might be too many parameters for this method, would it be better to make an `initUniswap` method for these 3 `_uniswap` parameters?